### ISSUE TITLE
Example of setting up merlin-auto-hl

### ISSUE THREAD

**Issue by kuwze** [Khady/merlin-eldoc#1]
I just wanted to add that this is how I setup `merlin-auto-hl` (after cloning this repository into `~/.emacs.d/bootleg/` on my system):

```emacs-lisp
(use-package merlin-auto-hl
  :load-path "~/.emacs.d/bootleg/merlin-auto-hl/"
  :hook
  (reason-mode . merlin-hl-identifier-mode)
  (tuareg-mode . merlin-hl-identifier-mode)
  (caml-mode-hook . merlin-hl-identifier-mode))
```

There seems to be an issue with using quelpa:
`(quelpa '(merlin-auto-hl :repo "Khady/merlin-auto-hl" :fetcher github))`
I get the following error:
> Debugger entered--Lisp error: (error "Package lacks a file header")
>   signal(error ("Package lacks a file header"))
>   error("Package lacks a file header")

**Comment by Khady** [comment]
Thanks for the report. I will add the file header.

**Comment by Khady** [comment]
Fixed by 5e1e373. The quelpa issue is traked in #2.
